docs/cookbook/platform-sh.rst

```diff
@@ -242,7 +248,13 @@ When you get connected please run:
 
 .. code-block:: bash
 
-    $ php bin/console sylius:install
+    $ php bin/console sylius:install --env prod
+
+.. warning::
+
+    By default platform.sh creates only one instance of database with the `main` name. Installing Sylius on dev environment would required
+`main_dev` which of course will end up with an exception. Deploying development environment is possible but will require a little bit more
```

Comment:
I think it's good to note that Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch. It's rather hard to activate per branch prod/dev settings in my experience until now using Platform.sh.